JavScript/script.js

```diff
@@ -28,4 +29,63 @@ mobMenuLinks.forEach((link) => {
     mobileMenu.classList.toggle('hidden');
     body.style.overflow = 'auto';
   });
+});
+
+// Create Javascript Objects for Teacher Section
+
+const teacher1 = {
+  image: 'images/yoga_woman_square.jpg',
+  alt: 'Female Yoga Teacher meditating',
+  name: 'Jane Doe',
+  subtitle: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit.',
+  description: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient.',
+};
+
+const teacher2 = {
+  image: 'images/yoga_man_square.jpg',
+  alt: 'Male Yoga Teacher meditating',
+  name: 'John Doe',
+  subtitle: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit.',
+  description: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient.',
+};
+
+const teacher3 = {
+  image: 'images/massage_square.jpg',
+  alt: 'Massage therapist',
+  name: 'Levi Smith',
+  subtitle: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit.',
+  description: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient.',
+};
+
+const teacher4 = {
+  image: 'images/chef_square.jpg',
+  alt: 'Female chef portrait',
+  name: 'Amy Meyer',
+  subtitle: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit.',
+  description: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient.',
+};
+
+const teachers = [teacher1, teacher2, teacher3, teacher4];
+
+// Function to create HTML
+
+const teachersHtmlTemplate = (obj) => `<img class="teacherImage" src="${obj.image}" alt="${obj.alt}"/>
+  <div class="teacherCont">
+    <h4 class="teacherName">${obj.name}</h4>
+    <h5 class="teacherSubtitle">${obj.subtitle}</h5>
+    <p class="teacherDescription">${obj.description}</p>
+  </div>`;
+
+// Append template
+
+document.addEventListener('DOMContentLoaded', () => {
+  teachers.forEach((teacher) => {
+    const dinContent = document.createDocumentFragment();
+    const newLi = document.createElement('li');
+    // newLi.className = 'works_card';
+
+    newLi.innerHTML = teachersHtmlTemplate(teacher);
+    dinContent.appendChild(newLi);
+    teachersCont.appendChild(dinContent);
+  });
 });
```

Comment:
Added the footer, however, I would recommend adding `overflow-x: hidden` in your CSS to fix the overflow of the page.
![html10](https://user-images.githubusercontent.com/55635977/142261578-47d691c3-7256-444c-9863-2847743a8657.png)

Reply:
Thank you Emmyn, 
I have added an overflow-x: hidden to the body element, which successfully removed the side scroll: 

<img width="1374" alt="Screen Shot 2021-11-17 at 16 33 47" src="https://user-images.githubusercontent.com/75140420/142293255-1447d9ce-2ab3-4996-b2cb-c79ec33c112a.png">